addons/binding/org.openhab.binding.dscalarm/ESH-INF/thing/it100bridge.xml

```diff
@@ -45,4 +49,10 @@
 		<description>Reset Switch</description>
 	</channel-type>
 
+    <channel-type id="command">
```

Comment:
You must define channel types only once within a binding, i.e. there must not be a channel type definition per bridge xml file.